.github/workflows/release_builds.yml

```diff
@@ -0,0 +1,51 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-alpha.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-beta.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-rc.[0-9]+'
```

Comment:
Do we really want to use `-sumo-[0-9]+` suffix for those images?

Reply:
I think that they are not needed.